helm/ingress-azure/templates/configmap.yaml

```diff
@@ -78,7 +78,7 @@ data:
 {{- end }}
 
 {{- if .Values.armAuth -}}
-{{- if eq .Values.armAuth.type "aadPodIdentity"}}
+{{- if or (eq .Values.armAuth.type "aadPodIdentity") (eq .Values.armAuth.type "workloadIdentity") }}
```

Comment:
do we really need this for workload identity ? `AZURE_CLIENT_ID` is dynamically injected by workload identity webhook mutating controller and not sure what the other variable is used for

Reply:
`USE_MANAGED_IDENTITY_FOR_POD` is required to bypass the cloud provider config based authorizer.

`AZURE_CLIENT_ID` - yes, this is automatically injected. But I keep it as this doesn't do any harm.